Doc/library/annotationlib.rst

```diff
@@ -204,12 +204,6 @@ Classes
       means may not have any information about their scope, so passing
       arguments to this method may be necessary to evaluate them successfully.
 
-      .. important::
-
-         Once a :class:`~ForwardRef` instance has been evaluated, it caches
-         the evaluated value, and future calls to :meth:`evaluate` will return
-         the cached value, regardless of the parameters passed in.
-
```

Comment:
should we also add a `.. versionchanged` note to the typing docs at https://docs.python.org/3.14/library/typing.html#typing.ForwardRef about this change?

Reply:
I can add something there, but note the caching behavior was never documented in the first place; I added this note when I wrote annotationlib.

Reply:
I know it was never documented, but I still feel like explicitly documenting that something changed might be nice :-)

Reply:
Fair enough!